src/components/upload/FileDropZone.js

```diff
@@ -1,27 +1,28 @@
+import { color } from "@mui/system";
 import DropZone from "react-dropzone";
-import styled from "styled-components";
+import {styled} from '@mui/system';
 
-const DropZoneRoot = styled.div`
-  width: 100%;
-  height: 200px;
-  border: 4px dashed #AFAFAF;
-  border-radius: 16px;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  align-items: center;
-  cursor: pointer;
-  padding: 18px;
-  
-  :hover {
-    background-color: #F8F8F8;
-    border: 4px solid #AFAFAF;
-  }
-`;
+const DropZoneRoot = styled('div')({
+     width : '100%',
+     height: 200,
```

Comment:
```suggestion
     height: '200px',
```
We should give units here for clarity.